### PR TITLE
intmdb 614 network container profile changes

### PR DESCRIPTION
**Pull request by govardhanpagidi**
## Description

Please include a summary of the fix/feature/change, including any relevant motivation and context.

Link to any related issue(s): 

## Type of change:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

**Comment by govardhanpagidi** [comment]
VpcId param may not have value all the time when we do GET Container Operation, hence not considered for ReadOnly Property, otherwise stack will fail when it doesn't have any value to return.

File: cfn-resources/network-container/docs/README.md [hunk 1]
```diff
@@ -69,19 +93,17 @@ _Type_: String
 
 _Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
 
-#### ApiKeys
-
-_Required_: Yes
+#### Profile
 
-_Type_: <a href="apikeydefinition.md">apiKeyDefinition</a>
+Profile used to provide credentials information, (a secret with the cfn/atlas/profile/{Profile}, is required), if not provided default is used
```
Comment (colm-quinn) [inline cfn-resources/network-container/docs/README.md:98]:
```suggestion
The profile is defined in AWS Secret manager. See [Secret Manager Profile setup](../../../examples/profile-secret.yaml).
```

File: cfn-resources/network-container/mongodb-atlas-networkcontainer.json [hunk 2]
```diff
@@ -40,39 +26,56 @@
             "description": "Unique 24-hexadecimal digit string that identifies the network peering container.",
             "type": "string"
         },
-        "ApiKeys": {
-            "$ref": "#/definitions/apiKeyDefinition"
+        "Profile": {
+            "type": "string",
+            "description": "Profile used to provide credentials information, (a secret with the cfn/atlas/profile/{Profile}, is required), if not provided default is used",
```
Comment (colm-quinn) [inline cfn-resources/network-container/mongodb-atlas-networkcontainer.json:31]:
```suggestion
            "description": "The profile is defined in AWS Secret manager. See [Secret Manager Profile setup](../../../examples/profile-secret.yaml).",
```

File: cfn-resources/network-container/test/inputs_1_create.template.json [hunk 3]
```diff
@@ -1,9 +1,6 @@
 {
   "ProjectId": "$ATLAS_PROJECT_ID",
-  "ApiKeys" : {
-    "PublicKey" : "$ATLAS_PUBLIC_KEY",
-    "PrivateKey" : "$ATLAS_PRIVATE_KEY"
-  },
+  "Profile" : "default",
   "RegionName": "US_EAST_1",
   "AtlasCidrBlock": "10.8.0.0/21"
 }
```
Comment (colm-quinn) [inline cfn-resources/network-container/test/inputs_1_create.template.json:6]: When making the other changes, can we add in the EOL here as well - thanks!

**Review by colm-quinn — APPROVED** [review]
LGTM, thanks for the updates!

**Pull request merged**